config/crd/v1alpha2/gateway.networking.k8s.io_httproutes.yaml

```diff
@@ -1219,6 +1289,7 @@ spec:
                             detached from the Gateway. \n Support: Core"
                           maxLength: 253
                           minLength: 1
+                          pattern: ^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$
                           type: string
```

Comment:
What is the largest CRD yet? We should register in that competition if one exists.